sgkit/stats/aggregation.py

```diff
@@ -327,6 +327,77 @@ def count_cohort_alleles(
     return conditional_merge_datasets(ds, new_ds, merge)
 
 
+def infer_call_allele_frequencies(
```

Comment:
``infer`` feels like the wrong word here, but I see you're following the established practices so I'm not sure.

@tomwhite, any thoughts?

This is a nice well-defined operation that we could pull out in a separate PR to make this one a bit lighter - what do you think @timothymillar?

Reply:
Agreed (I think I started the use of infer...). I'm not sure how people feel about having a method `call_allele_frequencies` which returns variable `call_allele_frequency`? There seems to be a pattern of plural for the method and singular for the variable.

There is an unexposed method `allele_frequency` which calculates `variant_allele_frequency`.  We could potentially rename this method to `variant_allele_frequencies` to match.

Happy to start a separate PR.

Reply:
No strong thoughts on this one. I'd be fine with functions called `call_allele_frequencies` and `variant_allele_frequencies`.